.vscode/settings.json

```diff
@@ -3,6 +3,6 @@
          "watch": true,
     },
     "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
+        "source.fixAll.eslint": "explicit"
```

Comment:
Is it intended? Why we do this?

Reply:
My version of VS Code changed these settings on its own. I will now revert back to the previous version.